homeassistant/components/ping/device_tracker.py

```diff
@@ -52,6 +56,7 @@ async def async_setup_scanner(
                     CONF_NAME: dev_name,
                     CONF_HOST: dev_host,
                     CONF_PING_COUNT: config[CONF_PING_COUNT],
+                    CONF_CONSIDER_HOME: config[CONF_CONSIDER_HOME],
```

Comment:
I am not seeing this option in the platform schema, was it removed in an earlier change?

Reply:
It is included in `BASE_PLATFORM_SCHEMA`, which is extended in line 33.

Reply:
Ah okay